Source: Franz Dietrich
Upstream: No
Reason: create the score directory.
--- a/Makefile.am
+++ b/Makefile.am
@@ -100,8 +100,8 @@
 
 if UNIX_SCOREDIR
 
-install-data-hook:	$(DESTDIR)$(kobo_scoredir)
-	chmod a+w $(DESTDIR)$(kobo_scoredir)
+install-data-hook:
+	-mkdir -m 774 $(DESTDIR)$(kobo_scoredir)
 
 else

